test/browserify-test.js

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2012-2016 LevelUP contributors
+ * See list at <https://github.com/level/levelup#contributing>
+ * MIT License <https://github.com/level/levelup/blob/master/LICENSE.md>
+ */
+
+var common  = require('./common')
+
+  , assert  = require('referee').assert
+  , refute  = require('referee').refute
+  , buster  = require('bustermove')
+  , browserify = require('browserify')
+  , path = require('path')
+
+var PACKAGE_JSON = path.resolve(path.join(__dirname, '..', 'package.json'))
```

Comment:
I think there's no need for the `path.resolve()` here, `path.join` with `__dirname` will give you an absolute path already